src/containers/change-password/ChangePassword.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import { Button, IconLock } from '@audius/stems'
+import cn from 'classnames'
+import { useDispatch, useSelector } from 'react-redux'
+
+import LoadingSpinnerFullPage from 'components/loading-spinner-full-page/LoadingSpinnerFullPage'
+import EnterPassword from 'components/sign-on/EnterPassword'
+import { Status } from 'store/types'
+
+import styles from './ChangePassword.module.css'
+import { ConfirmCredentials } from './ConfirmCredentials'
+import { getChangePasswordStatus, getCurrentPage } from './store/selectors'
+import { changePage, changePassword, Page } from './store/slice'
+
+const messages = {
+  helpTexts: [
+    'Please enter your email and current password.',
+    'Create A New Password That Is\n Secure And Easy To Remember!',
+    'Changing Password, Please wait',
+    'Your Password Has Been Changed',
+    <span key={Page.FAILURE} className={styles.error}>
```

Comment:
this is an interesting strategy. not entirely sure how much I like it -- maybe better to pull these out into strings and leave the layout for the component. the high level reason we pull messages out into a const is because someday we will probably want to i18n them

Reply:
Yeah I don't love it either. I'll revisit